src/state/models/ui/create-account.ts

```diff
@@ -117,6 +118,8 @@ export class CreateAccountModel {
       this.setIsProcessing(false)
       this.setError(cleanError(errMsg))
       throw e
+    } finally {
```

Comment:
Moved a lot of the tracking code into a `finally` block in `try-catch`, as well as moving it closer to the API call, regardless of which component calls it.

This was done because previously we were missing a lot of data because stuff like `FeedItem:Like` was only called for FeedItems and not Post or ThreadItems which can also execute the `like` function. I did this in a backwards-compatible manner, so all the original `track` calls are still there and can be used but I added additional `track` calls near the source for better tracking.